vendors/circom/prover_main.cc

```diff
@@ -123,8 +123,8 @@ void CreateProof(const base::FilePath& zkey_path,
     math::IcicleNTTOptions ntt_options;
     ntt_options.fast_twiddles_mode = !options.disable_fast_twiddles_mode;
     CHECK(icicle_ntt_holder->Init(domain->group_gen(), ntt_options));
+    domain->set_icicle(&icicle_ntt_holder);
```

Comment:
Are you trying to disable benchmark with CUDA for domain that has size less than 8?

Reply:
Yes, it takes forever in that case.